ingestion/src/metadata/ingestion/source/dynamodb.py

```diff
@@ -77,18 +78,22 @@ def ingest_tables(self, next_tables_token=None) -> Iterable[OMetaDatabaseAndTabl
         tables = list(self.dynamodb.tables.all())
         for table in tables:
             try:
-                if not self.config.table_filter_pattern.included(table.name):
+                if (
+                    table.name
+                    not in self.config.sourceConfig.config.tableFilterPattern.includes
+                ):
                     self.status.filter(
                         "{}".format(table.name),
                         "Table pattern not allowed",
                     )
                     continue
                 database_entity = Database(
-                    name=self.config.db_name,
+                    id=uuid.uuid4(),
+                    name="default",
                     service=EntityReference(id=self.service.id, type="databaseService"),
                 )
 
-                fqn = f"{self.config.service_name}{FQDN_SEPARATOR}{database_entity.name}{FQDN_SEPARATOR}{table}"
+                fqn = f"{self.config.serviceName}{FQDN_SEPARATOR}{database_entity.name}{FQDN_SEPARATOR}{table}"
```

Comment:
Do not inform the FQN. It is optional in `Table` and the backend will regenerate it